From 9974dabeec3370797ee73a30f86bdb5d522389c4 Mon Sep 17 00:00:00 2001
From: Chris Dimich <chris.dimich@ezurio.com>
Date: Thu, 16 Jan 2025 13:09:55 -0800
Subject: [PATCH 1/1] Makefile: update CFLAGS

Fix the following errors:

- Remove -Werror:

cc1: all warnings being treated as errors

- add -I${STAGING_INCDIR_AT}

fatal error: stdio.h: No such file or directory

- add --sysroot=${STAGING_DIR_TARGET_AT}

cannot find Scrt1.o: No such file or directory
cannot find crti.o: No such file or directory
cannot find crtbeginS.o: No such file or directory
...

Signed-off-by: Chris Dimich <chris.dimich@ezurio.com>
---
 Makefile | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 954cb19..5123eb3 100644
--- a/Makefile
+++ b/Makefile
@@ -3,9 +3,11 @@ VERSION	= 0.9.1
 PREFIX	= /usr
 
 CC	= gcc
-CFLAGS	= -W -Wall -Wextra -Werror \
+CFLAGS	= -W -Wall -Wextra \
 	-DVERSION=\"$(VERSION)\" \
-	-g
+	-g \
+	-I${STAGING_INCDIR_AT} \
+	--sysroot=${STAGING_DIR_TARGET_AT}
 LDFLAGS =
 
 ifeq "REMOVE_THIS_FOR_RELEASE" "REMOVE_THIS_FOR_RELEASE"
-- 
2.47.1

